tests/examples/tokens/test_erc1155ownable_ETHtester.py

```diff
@@ -0,0 +1,316 @@
+import pytest
+# ERC1155 ownable, opensea compatible tests
+# @author Dr. Pixel (github: @Doc-Pixel)
+
+# constants - contract deployment
+CONTRACT_NAME = 'TEST 1155'
+CONTRACT_SYMBOL= 'T1155'
+CONTRACT_URI = 'https://mydomain.io/NFTdata/{id}'
+NEW_CONTRACT_URI = 'https://mynewdomain.io/NFTdata/{id}'
+ERC165_INTERFACE_ID = '0x01ffc9a7'
+ERC1155_INTERFACE_ID = '0xd9b67a26'
+ERC1155_INTERFACE_ID_METADATA = '0x0e89341c'
+ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
+
+# minting test lists
+mintBatch = [1,2,3,4,5,6,7,8,9,10]
+mintBatch2 = [11,12,13,14,15,16,17,19,19,20]
+minBatchSetOf10 = [1,1,1,1,1,1,1,1,1,1]
+mintConflictBatch = [1,2,3]
+
+@pytest.fixture
+def NFT_contract(get_contract, w3):
```

Comment:
Can this fixture just be renamed to `erc1155`? Would make it easier to follow and also not mixed case

Reply:
renamed to erc1155